source/extensions/filters/network/mongo_proxy/codec_impl.cc

```diff
@@ -355,6 +355,11 @@ bool DecoderImpl::decode(Buffer::Instance& data) {
   }
 
   data.drain(sizeof(int32_t));
+  if (message_length < Message::MessageHeaderSize) {
+    ENVOY_LOG(error, "message size {} less than min. message size {}", message_length,
```

Comment:
In general we try not to log info and above in the data path as it will log spam. I think debug is fine for this. Thank you!

/wait

Reply:
Changed to debug. Thank you for the fast review.